application/src/main/java/org/thingsboard/server/service/install/DefaultDatabaseSchemaSettingsService.java

```diff
@@ -88,17 +86,28 @@ public String getPackageSchemaVersion() {
     @Override
     public String getDbSchemaVersion() {
         if (schemaVersionFromDb == null) {
-            Long version = getSchemaVersionFromDb();
-            if (version == null) {
+            Long dbVersion = getSchemaVersionFromDb();
+            if (dbVersion == null) {
                 onSchemaSettingsError("Upgrade failed: the database schema version is missing.");
             }
 
             @SuppressWarnings("DataFlowIssue")
-            long major = version / 1000000;
-            long minor = (version % 1000000) / 1000;
-            long patch = version % 1000;
-
-            schemaVersionFromDb = major + "." + minor + "." + patch;
+            long version = dbVersion;
+
+            if (version < 1000000000) {
```

Comment:
Could you please use the following syntax in this method for better comprehension: 1_000_000_000